myst_nb/_static/mystnb.css

```diff
@@ -36,6 +36,7 @@ div.cell_input > div, div.cell_output div.output > div.highlight {
   margin-top: 1em;
 }
 
+/* TODO remove/replace? */
```

Comment:
Seems like we can replace this if we are removing the dependency on jupyter_sphinx

Reply:
removed